website/content/en/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines.md

```diff
@@ -10,10 +10,91 @@ These components integrate SageMaker with the portability and orchestration of K
 
 ## Available components
 
-You can create a Kubeflow Pipeline built entirely using SageMaker components, or integrate individual components into your workflow as needed. Available Amazon SageMaker components can be found in the [Kubeflow GitHub repository](https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker).
+You can create a Kubeflow Pipeline built entirely using SageMaker components, or integrate individual components into your workflow as needed. Available Amazon SageMaker components can be found in the [Kubeflow Pipelines GitHub repository](https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker).
 
-## Tutorials
+Currently, we support two versions of these SageMaker components - the traditional v1 components and the ACK Based v2 components. The Sample included with Kubeflow is currently based off of the v2 Training Component. However, you can just as easily configure and use all others by referring to our more detailed guides over at the pipelines repo - 
+1. [AWS Samples for SageMaker Kubeflow Pipeline Components](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples)
```

Comment:
lets not mention these docs right now, these instructions are specific to standalone installation so it will be confusing. Instead I would suggest making this an end to end tutorial with instructions needed to run for running v1 components as well.

Add this info: The pipeline runs are executed in user namespaces using the `default-editor` Kubernetes service account.

Reply:
We can add that once we have a second sample pipeline